src/styles/index.scss

```diff
@@ -1,3 +1,76 @@
+$main-color: #eee;
+$hover-color: #ffbfcb;
+$border-color: #000;
+$day-size: 100px;
+$border-width: 1px;
+$gap: 1px;
+$padding: 10px;
+$font-size: 30px;
```

Comment:
Better move it to a separate file. 
Also, check naming. You need to name your variable clearly.